### PR TITLE
Add "adopt a module" message

### PR DESCRIPTION
**Pull request by eLBati**
See https://github.com/OCA/oca-github-bot/issues/93

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
thanks ! 

could you check travis ? 

kind regards.

**Comment by sbidoul** [comment]
@eLBati thanks! That was quick :+1: 

A few comments:

- A link to the maintainer role definition page would be useful in the comment. It should be configurable (https://odoo-community.org/page/maintainer-role by default).
- This feature should be behind a configuration toggle as oca-github-bot is used outside of OCA (there are many examples)
- Be careful the mention could be empty if it's a maintainer that does the PR so the implementation is probably a little bit more complex and may require some minor refactoring

**Comment by eLBati** [comment]
@sbidoul I made the changes. Please review. Thanks

File: src/oca_github_bot/tasks/mention_maintainer.py
Comment (sbidoul) [inline src/oca_github_bot/tasks/mention_maintainer.py:89]:
```suggestion
            "If you would like to adopt it, read the [maintainer role description](%s) \n"
            "and add your GitHub name to `maintainers` key of `__manifest__.py`"
            " file!"
```

File: environment.sample
Comment (sbidoul) [inline environment.sample:11]:
```suggestion
# Maintainer role page URL, used in "adopt a module" message.
# Set this to a URL that describes the maintainer role to enable the message.
#MAINTAINER_ROLE_PAGE=
```

File: src/oca_github_bot/config.py
Comment (sbidoul) [inline src/oca_github_bot/config.py:52]:
```suggestion
MAINTAINER_ROLE_PAGE = os.environ.get("MAINTAINER_ROLE_PAGE")
```

For compatibility with existing deployments.

**Review by sbidoul — COMMENTED** [review]
Hi Lorenzo, sorry for the delay. I made a couple of suggestions to keep backward compatibility and tweak the mention a little bit. Otherwise this looks good.

**Comment by sbidoul** [comment]
Oh, and I notice these maintainer mentions does not seem to be mentioned in the readme. I can add them after merging.

**Comment by eLBati** [comment]
@sbidoul changes applied, thanks

**Comment by sbidoul** [comment]
@eLBati I tweaked a little bit in #139 and fixed the tests. Now I think there is one last thing to do: when a maintainer adds him/herself to an addon, the adopt mention must not appear.

**Pull request merged**
